hips/utils/tests/test_healpix.py

```diff
@@ -22,3 +24,19 @@ def test_boundaries():
     radec_precomp = [[264.375,  258.75,  264.375,  270.],
                      [-24.624318,  -30.,  -35.685335,  -30.]]
     assert_allclose([radec.ra, radec.dec], radec_precomp)
+
+def test_compute_healpix_pixel_indices():
+    order = 3
+    nside = hp.order2nside(order)
+
+    skycoord = SkyCoord(10, 20, unit="deg")
+    wcs_geometry = WCSGeometry.create(skycoord, (10, 20), 'CEL', 'AIT', 1.0, 1)
+
+    """
+    These pixel values were obtained for the all-sky image located at:
+    https://github.com/gammapy/gammapy-extra/blob/master/datasets/catalogs/fermi/gll_psch_v08.fit.gz?raw=true
+    """
+    pixels_precomp = [0, 767]
+
+    pixels = compute_healpix_pixel_indices(wcs_geometry, nside)
+    assert_allclose([pixels[0], pixels[-1]], pixels_precomp)
```

Comment:
Can you add an assert on `len(pixels)`? How many are there now for this example you have.
You could also change the example so that you have ~ 5 to 10 pixels, and then just do the `assert_allclose` on all of those.
I was only objecting to the ~ 700 pixels long list you had before, because it made the test hard to read.